sdktests/custom_matchers_events.go

```diff
@@ -23,10 +23,18 @@ func EventHasKind(kind string) m.Matcher {
 	return m.JSONProperty("kind").Should(m.Equal(kind))
 }
 
-func HasContextKeys(user lduser.User) m.Matcher {
-	return m.JSONProperty("contextKeys").Should(m.MapOf(
-		m.KV("user", m.Equal(user.GetKey())),
-	))
+func HasContextKeys(context ldcontext.Context) m.Matcher {
+	var kvs []m.KeyValueMatcher
+	if context.Multiple() {
```

Comment:
There aren't yet any tests that actually use a multi-kind context, but when we do, this logic will apply.